orchestra/static/orchestra/timing/timecard/controller.js

```diff
@@ -74,7 +75,16 @@
           return;
         }
         entry.description = entry.editData.description;
```

Comment:
i get a TypeError: Cannot read property 'description' of undefined at this line when i try to save an empty entry